bisection_script.py

```diff
@@ -25,9 +25,13 @@ def parse_args(argv=None):
                         dest="changeset",
                         type=str,
                         help="Current changeset.")
+    parser.add_argument("-p", "--patch",
+                        dest="patch",
+                        type=str,
+                        help="Topmost additional patch in series to push before trypatch.")
     parser.add_argument("--cwd",
                         dest="cwd",
-                        #required=True,
+                        required=True,
```

Comment:
Actually required is necessary, since the commands below will fail to apply if cwd of inbound is not given. I meant to put it in previous patch, just forgot to uncomment it.